css-typed-om/Overview.bs

```diff
@@ -784,10 +782,20 @@ The following are the arithmetic operations you can perform on dimensions:
         is set to |this|.
 </div>
 
-<div class=issue>
-    Define {{CSSNumericValue/equals()}}.
+<div algorithm="CSSNumericValue.equals(...|value|)">
+    The <dfn for=CSSNumericValue method>equals(...|value|)</dfn> method,
+    when called on a {{CSSNumericValue}} |this|,
+    must perform the following steps:
+
+    1. If the |value| item is the same type as the {{CSSNumericValue}} the method is being called on continue to step 2, else return false.
```

Comment:
* Just reverse this conditional and return false. No need to explicitly say to go to the next step; that's implied by the list.  
* Be clear what you mean by "type" - you use the exact same word in the second step, but clearly mean something different.  I suspect here you're talking about the interface type.
* "the CSSNumericValue the method is being called on" is already stored in the `|this|` variable, so you just refer to it that way.